lib/appliance_console/certificate_authority.rb

```diff
@@ -87,18 +85,10 @@ def configure_pgserver
       self.pgserver = cert.status
     end
 
-    def configure_api
-      Certificate.new(
-        :cert_filename => "#{CFME_DIR}/apiclient.crt",
-        :root_filename => "#{CFME_DIR}/root.crt",
-        :service       => "manageiq",
-        :extensions    => %w(client),
-        :ca_name       => ca_name,
-        :hostname      => hostname,
-        :owner         => "apache.apache"
-      ).request
+    def configure_http
       cert = Certificate.new(
-        :cert_filename => "#{CFME_DIR}/apiserver.crt",
+        :key_filename  => "#{CFME_DIR}/server.cer.key",
+        :cert_filename => "#{CFME_DIR}/server.cer",
```

Comment:
we currently store these as `server.cer.key` and `server.cer`.
That is in the applinace